.travis.yml

```diff
@@ -18,18 +18,9 @@ addons:
 
 after_success:
  - ./extra/make-testcoveralls.sh
-
-before_deploy:
+ # below is the placeholder to leverage TravisCI to verify building capability is good. will be replaced by npm package build.
```

Comment:
personally, I think if it's not for coverage or generate any packages for RackHD, don't need to verify the building process here, just do it in the build places.